docs/section22/Rule22-3.md

```diff
@@ -0,0 +1,46 @@
+
+# CHW&CW - Rule 22-3  
+
+**Rule ID:** 22-3  
+**Rule Description:** For Baseline chilled water loop that is not purchased cooling, chilled-water supply temperature shall be reset based on outdoor dry-bulb temperature.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.9 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13
```

Comment:
Please remove system 11 from the list because it is subject to Exception 1 to G3.1.3.9 which we are covering in Rule 22-5